### PR TITLE
docs(optimize): update screenshots

### PR DESCRIPTION
**Pull request by misiekhardcore**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: optimize/components/userguide/process-analysis/report-analysis/img/NumberConfiguration.png
Comment (OmranAbazid) [inline optimize/components/userguide/process-analysis/report-analysis/img/NumberConfiguration.png:1]: The popover seems to be truncated here not showing the input fuly

File: optimize/components/userguide/img/dashboard-dashboardEditActions.png
[the remark this answers is not in the document]
Comment (misiekhardcore) [reply optimize/components/userguide/img/dashboard-dashboardEditActions.png:1]: the problem is that I cant see any process that shows any data in this dashboard

File: optimize/components/userguide/process-analysis/report-analysis/img/NumberConfiguration.png
Comment (misiekhardcore) [reply optimize/components/userguide/process-analysis/report-analysis/img/NumberConfiguration.png:1]: fixed now

**Review by christinaausley — APPROVED** [review]
I noticed the "user" icon in the top right corner is occasionally cut off -- this isn't blocking, but I'm wondering where the inconsistency is coming from.

Is this ready to merge now, or upon `8.4` release?

File: optimize/components/userguide/img/dashboard-dashboardEditActions.png
[the remark this answers is not in the document]
Comment (misiekhardcore) [reply optimize/components/userguide/img/dashboard-dashboardEditActions.png:1]: I tried different templates and with our data all of them seem to be empty

**Comment by misiekhardcore** [comment]
@christinaausley this is really interesting, I haven't noticed that the icon is cut before. I will take a note to check that.
I think we can merge it already.

**Comment by misiekhardcore** [comment]
FYI I made [this ticket](https://jira.camunda.com/browse/OPT-7524) to look into this

**Pull request merged**